src/middlewared/middlewared/utils/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class DeprecatedFile(Exception):
```

Comment:
Bad name, because in our case it's not deprecated, it just should not exist. Maybe `ShouldNotExist` or something like that?